microsoft-edge/puppeteer/index.md

```diff
@@ -3,109 +3,115 @@ description: Use Puppeteer to automate and test in Microsoft Edge.
 title: Puppeteer overview
```

Comment:
Note for the future: it makes it easier to review PRs when they're focused on just one thing. In this case: an editor pass on PWA articles.
Updating this Puppeteer article feels a bit out of scope. It makes reviewing the PR harder for me. I don't have a lot of knowledge at all about Puppeteer, so I don't really feel comfortable signing off on these changes.
That said, you are not changing content here, so it's fine. But I think in the future it would help us to keep PRs as narrowly focused as we can.